Webscraper/webscraper.py

```diff
@@ -173,8 +199,8 @@
                             abstracts.append('')
                     entry_temp = ''
                     n_combines = 0
-                
-                
+        else:
+            print('Post after Jinger, skpping')
```

Comment:
Maybe adjust name of `jinger_broke_it` variable... ;)

Reply:
FINE